<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>454</width>
    <height>413</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLineEdit" name="inputInputFolder">
      <property name="inputMask">
       <string notr="true"/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>Input folder</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="butBrowse">
      <property name="text">
       <string>&amp;Browse...</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="output1">
       <attribute name="title">
        <string>Tab 1</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_7">
        <item row="3" column="0">
         <widget class="QLineEdit" name="inputOutputFolder1">
          <property name="placeholderText">
           <string>Output folder</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0" alignment="Qt::AlignHCenter">
         <widget class="QPushButton" name="butBrowseOutput1">
          <property name="text">
           <string>Browse...</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0" alignment="Qt::AlignHCenter">
         <widget class="QCheckBox" name="enable1">
          <property name="text">
           <string>Enable</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QFrame" name="frame_6">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QGridLayout" name="gridLayout_8">
           <item row="0" column="0">
            <widget class="QCheckBox" name="resize1">
             <property name="text">
              <string>Resize</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="inputResize1"/>
           </item>
           <item row="1" column="0">
            <widget class="QCheckBox" name="rename1">
             <property name="text">
              <string>Rename</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="inputRename1"/>
           </item>
           <item row="2" column="0">
            <widget class="QCheckBox" name="watermark1">
             <property name="text">
              <string>Watermark</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLineEdit" name="inputWatermark1"/>
           </item>
           <item row="2" column="2">
            <widget class="QPushButton" name="butBrowseWatermark1">
             <property name="text">
              <string>Browse...</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="output2">
       <attribute name="title">
        <string>Page</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_13">
        <item row="0" column="0" alignment="Qt::AlignHCenter">
         <widget class="QCheckBox" name="enable2">
          <property name="text">
           <string>Enable</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLineEdit" name="inputOutputFolder2">
          <property name="placeholderText">
           <string>Output folder</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0" alignment="Qt::AlignHCenter">
         <widget class="QPushButton" name="butBrowseOutput2">
          <property name="text">
           <string>Browse...</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QFrame" name="frame_7">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QGridLayout" name="gridLayout_14">
           <item row="0" column="0">
            <widget class="QCheckBox" name="resize2">
             <property name="text">
              <string>Resize</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="inputResize2"/>
           </item>
           <item row="1" column="0">
            <widget class="QCheckBox" name="rename2">
             <property name="text">
              <string>Rename</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="inputRename2"/>
           </item>
           <item row="2" column="0">
            <widget class="QCheckBox" name="watermark_2">
             <property name="text">
              <string>Watermark</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLineEdit" name="inputWatermark2"/>
           </item>
           <item row="2" column="2">
            <widget class="QPushButton" name="butBrowseWatermark2">
             <property name="text">
              <string>Browse...</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="output3">
       <property name="accessibleName">
        <string/>
       </property>
       <attribute name="title">
        <string>Page</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_15">
        <item row="0" column="0" alignment="Qt::AlignHCenter">
         <widget class="QCheckBox" name="enable3">
          <property name="text">
           <string>Enable</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLineEdit" name="inputOutputFolder3">
          <property name="placeholderText">
           <string>Output folder</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0" alignment="Qt::AlignHCenter">
         <widget class="QPushButton" name="butBrowseOutput3">
          <property name="text">
           <string>Browse...</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QFrame" name="frame_8">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QGridLayout" name="gridLayout_16">
           <item row="0" column="0">
            <widget class="QCheckBox" name="resize3">
             <property name="text">
              <string>Resize</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="inputResize3"/>
           </item>
           <item row="1" column="0">
            <widget class="QCheckBox" name="rename3">
             <property name="text">
              <string>Rename</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="inputRename3"/>
           </item>
           <item row="2" column="0">
            <widget class="QCheckBox" name="watermark3">
             <property name="text">
              <string>Watermark</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLineEdit" name="inputWatermark3"/>
           </item>
           <item row="2" column="2">
            <widget class="QPushButton" name="butBrowseWatermark3">
             <property name="text">
              <string>Browse...</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QProgressBar" name="progressBar">
      <property name="value">
       <number>24</number>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="butGo">
      <property name="text">
       <string>&amp;Go</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>454</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;Settings</string>
    </property>
    <addaction name="actionOpen_Profile"/>
    <addaction name="actionSave_Profile"/>
    <addaction name="separator"/>
    <addaction name="actionAbout"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionResize_help"/>
    <addaction name="actionRename_help"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen_Profile">
   <property name="text">
    <string>&amp;Open Profile...</string>
   </property>
  </action>
  <action name="actionSave_Profile">
   <property name="text">
    <string>&amp;Save Profile</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>&amp;Exit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>&amp;About...</string>
   </property>
  </action>
  <action name="actionResize_help">
   <property name="text">
    <string>Resize help...</string>
   </property>
  </action>
  <action name="actionRename_help">
   <property name="text">
    <string>Rename help...</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
